lib/services/ngsi/entities-NGSI-LD.js

```diff
@@ -65,6 +65,8 @@ function convertNGSIv2ToLD(attr) {
         // Properties
         case 'property':
         case 'string':
+        case 'text':
+        case 'textunrestricted':
```

Comment:
Should something about the changes done in this PR be mentioned in CHANGES_NEXT_RELEASE file?

Reply:
Fixed  04e5b8a